warpscript/utapi/sumField.mc2

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
Shouldn't this go in warpscript/util directory?

Reply:
It's already moved on my local copy :) I'll backport the change